src/AsyncStepper.hpp

```diff
@@ -220,6 +220,10 @@ class AsyncStepper
 		return TravelSteps;
 	}
 
+	void ResetAbsoluteStep() {
+		AbsoluteStep = 0;
+	}
+	

```

Comment:
consider:
```
void SetAbsoluteStep(long steps) { 
AbsoluteStep = steps; 
}
```

Reply:
I need to thing about what implies to change steps "on the fly". This steps are used in the internal state machine and in the acceleration and decceleration ramp.